sheet_to_triples/__main__.py

```diff
@@ -90,8 +91,8 @@ def parse_args(argv):
 def main(argv):
     args = parse_args(argv)
     runner = run.Runner.from_args(args)
-    if args.add_graph:
-        runner.graph.parse(args.add_graph, format='ttl')
+    for graph_to_add in args.add_graph or ():
+        runner.graph.parse(graph_to_add, format='ttl')
         runner.model['terms'] = []
```

Comment:
Does this make sense? We update the `terms` with each new graph/ontology/whatever via `rdf.update_model_terms`... and then on the next run through the loop we zero them out again. Or at least that's my current read of this.

Reply:
It does actually work, but yes, this makes a nonsense of the current logic. Will amend to not pointlessly repeat the terms replacing.

Reply:
Okay, so `runner.graph.parse` is additive (effectively appends to `runner.graph` one at a time) and the wiping of `terms` didn't actually matter because the full contents of `runner.graph` were being re-added each time. That makes more sense.